src/bci_build/package/kiosk.py

```diff
@@ -0,0 +1,165 @@
+"""Build descriptions for the containerized kiosk containers (X11+auxiliary
+apps).
+
+"""
+
+from pathlib import Path
+
+from bci_build.os_version import ALL_NONBASE_OS_VERSIONS
+from bci_build.os_version import CAN_BE_LATEST_OS_VERSION
+from bci_build.package import ApplicationStackContainer
+from bci_build.package import DevelopmentContainer
+from bci_build.package import ParseVersion
+from bci_build.package import Replacement
+from bci_build.package.helpers import generate_package_version_check
+from bci_build.package.versions import get_pkg_version
+
+_X11_FILES = {
+    "preferences": """ShowThemesMenu=0
+ShowLogoutMenu=0
+ShowFocusModeMenu=0
+QuickSwitch=0
+""",
+    "entrypoint.sh": (
+        (_x11_dir := (_pkg_dir := Path(__file__).parent) / "x11") / "entrypoint.sh"
+    ).read_text(),
+    "xinitrc": (_x11_dir / "xinitrc").read_text(),
+    "xorg.conf": (_x11_dir / "xorg.conf").read_text(),
+}
+
+X11_CONTAINERS = [
```

Comment:
can you please split that into a `x11.py` ? given that we have files in x11/ subdir that would be more aligned and easier to maintain going forward imho. 

also, not trying to be too nitpicky, but you're packaging xorg not X11 (that would be from the times before you were born..). can we just call it XORG instead?

Reply:
> can you please split that into a `x11.py` ? given that we have files in x11/ subdir that would be more aligned and easier to maintain going forward imho.

No, these three containers belong together imho

> 
> also, not trying to be too nitpicky, but you're packaging xorg not X11 (that would be from the times before you were born..). can we just call it XORG instead?

I have renamed it `xorg` instead of `x11`